v-next/hardhat-node-test-reporter/src/error-formatting.ts

```diff
@@ -57,10 +68,9 @@ ${stack}`;
 
 // TODO: Do this in a more robust way and that works well with windows
```

Comment:
is the TODO still needed?

Reply:
Yes, this is still a bit of a hack IMO. For example, see your later comment. Sometimes stack traces use file-urls, sometimes paths. Unclear when, and if we are properly handling everything. Eventually, we'll want to polish this, properly test it, etc.